xbmc/ContextMenuManager.cpp

```diff
@@ -72,7 +73,12 @@ void CContextMenuManager::Init()
       std::make_shared<CONTEXTMENU::CMarkWatched>(),
       std::make_shared<CONTEXTMENU::CMarkUnWatched>(),
   };
+
   ReloadAddonItems();
+
+  std::vector<std::shared_ptr<IContextMenuItem>> pvrItems = m_pvrMgr.GetContextMenuItems();
```

Comment:
Not sure what you had in mind with the CPVRContextMenuManager as it doesn't seem to do anything but hold the static items. I think it's fine to just include all the classes directly like above (for now, this whole part is a candidate for refactoring anyway). In a way they are 'easier' dependencies to deal with than the huge pvr manager.